### DISCUSSION TITLE
rust_sdk run_func pass params to wasmedge_bindgen

### DISCUSSION THREAD

**Discussion by ActivePeter** [WasmEdge/WasmEdge#3109] [Q&A]
<img width="529" alt="image" src="https://github.com/WasmEdge/WasmEdge/assets/8100696/3d2f8c6f-3fdb-4e16-9471-b9868e02b8fe">

I'm following the example https://github.com/second-state/wasmedge-rustsdk-examples/blob/0.4.0/wasmedge-bindgen/call-wasm-lib/Cargo.toml with wasmedge_sdk 0.11.0

I try to pass in the params docker::Param and got error
<img width="765" alt="image" src="https://github.com/WasmEdge/WasmEdge/assets/8100696/a5f16e62-76ef-41c1-9971-a83da10c0d0c">

<img width="286" alt="image" src="https://github.com/WasmEdge/WasmEdge/assets/8100696/4f446f2d-1955-4b50-9527-8288bbf2104a">

<img width="328" alt="image" src="https://github.com/WasmEdge/WasmEdge/assets/8100696/a6bd111d-e1de-4da7-ab1e-9526bdba80fb">

**Comment by hydai** [comment]
Hi @apepkuss 
Could you please check this issue?

**Comment by apepkuss** [comment]
@ActivePeter Sorry for the late reply. `wasmedge-rust-sdk` has been upgraded, so please refer to the examples in the `refactor/new_sdk` branch: https://github.com/second-state/wasmedge-rustsdk-examples/tree/refactor/new_sdk/wasmedge-bindgen

@L-jasmine Could you please help with this issue? Thanks a lot!
